pushsync/pusher.go

```diff
@@ -170,14 +170,8 @@ func (p *Pusher) sync() {
 				break
 			}
 
-			// increment synced count for the tag if exists
 			tag := item.tag
 			if tag != nil {
-				tag.Inc(chunk.StateSynced)
```

Comment:
We no longer increment synced tags on receive of receipts, but we still close the spans for individual chunks as soon as we get a receipt.